Ownership/Non-UK/Reykjavik.txt

```diff
@@ -1,19 +1,43 @@
-SECTOR:BIRD (000-245):0:24500
+SECTOR:BIRD East (085-345):8500:34500
```

Comment:
Class A in BIRD E seems to start at 055 not 085 - see AIP and LoA:
![image](https://user-images.githubusercontent.com/14115426/214679522-3d36e1c8-c95f-4bb2-a1c0-24b21550f470.png)
![image](https://user-images.githubusercontent.com/14115426/214679873-4c65156e-a867-4f35-8747-aa7ed803d6fc.png)